tests/platform_tests/daemon/test_pcied.py

```diff
@@ -37,7 +37,7 @@
 SIG_TERM = "-15"
 SIG_KILL = "-9"
 
-pcie_devices_status_tbl_key = "PCIE_DEVICES|status"
+pcie_devices_status_tbl_key = "PCIE_DEVICES"
```

Comment:
This key is before refactoring and the previous is the key after refactoring the pcied. We need to check either one of the key value pair exists.

Reply:
@sujinmkang fixed, please review